### DISCUSSION TITLE
Datatable multisort with optional metaKey, not default for multiple sort selection.

### DISCUSSION THREAD

**Discussion by rowanfuchs** [primefaces/community#3111] [PrimeVue]
Allow multi-sorting in Datatable without requiring the meta key by default. Using a meta key as the default for multi-sorting is considered a UX anti-pattern, as it often necessitates extra explanations to inform users on how to sort by multiple fields.

A issue reported earlier is here:
https://github.com/primefaces/primevue/issues/2716

The code in question would need an extra condition change with an additional property that toggles this functionality.
https://github.com/primefaces/primevue/blob/786f9c65e69c10e7d4df32f467a1bd895a44d926/packages/primevue/src/datatable/DataTable.vue#L562

**Comment by perlDreamer** [comment]
The Removable Sort feature in the DataTable seems like it would be a good route to take, since it would allow the user to deselect columns for sorting.
